### DISCUSSION TITLE
Why Filecoin Plus Should Remain Payment-Modality Agnostic

### DISCUSSION THREAD

**Discussion by lksbrssr** [filecoin-project/Allocator-Governance#258] [General]
## Why Filecoin Plus Should Remain Payment-Modality Agnostic
There has been some debate around whether Filecoin Plus hould require on-chain payments for storage deals. This post argues that Fil+ should remain neutral about payment modalities—off-chain or on-chain—because flexibility is key to driving adoption.

## The Role of Payments in Filecoin Plus
The purpose of Filecoin Plus is to attract quality data to the Filecoin network. A reasonable proxy for "quality" data is whether users are willing to pay for its storage. If data has value, its owners will invest in storing it. This willingness to pay signals that the service provides meaningful value. To facilitate payments for Filecoin storage services, there are fundamentally two methods:

1. On-chain payments: Payments made in stablecoins or tokens on the (Filecoin) blockchain
2. Off-chain payments: Traditional methods like invoices, wire transfers, or credit card payments

Both have distinct advantages and trade-offs.

## The Case for On-Chain Payments
On-chain payments offer several benefits:

- Transparency and Verification: On-chain payments are publicly verifiable, ensuring payment accountability and building trust between clients and providers.
- Network Usage: Transactions made using the Filecoin blockchain increase its usage and demand for blockspace, driving value across the ecosystem.
- Smart Contract Integration: On-chain payments enable developers to build automated solutions—such as smart contracts on the Filecoin Virtual Machine (FEVM)—for storage deals, unlocking innovation.
- Reinvestment into the Ecosystem: On-chain payments ensure that value created within the Filecoin ecosystem circulates internally rather than flowing out of it.

These factors make on-chain payments a desirable long-term goal for Filecoin adoption.

## The Case for Off-Chain Payments
Despite their advantages, on-chain payments may pose hurdles for many enterprise clients:

- User Familiarity: Most businesses are accustomed to conventional payment methods—like credit cards, invoices, or bank transfers. Requiring on-chain payments adds friction, making adoption less likely.
- Privacy: Many enterprises intentionally keep pricing information private. On-chain transparency, while beneficial in some cases, may be seen as a downside here.
- Adoption Friction: Filecoin is already introducing web3-native storage, which requires a behavioral shift. Adding mandatory web3-native payments creates an additional adoption barrier, reducing the pool of potential customers.
- Economic Value: Even if payments occur off-chain, they still generate economic inflows to Filecoin providers, offsetting operational costs and increasing sustainability. Fiat revenues indirectly support the Filecoin economy by funding network operations.

In short, flexibility with payment options lowers adoption barriers and aligns with how enterprises currently do business.

## Trade-Off: Adoption vs. Modality
The real debate isn’t on-chain payments vs. off-chain payments. It’s about adoption vs. non-adoption.

To grow Filecoin adoption, we must prioritize getting clients onto Filecoin-native storage services—regardless of how they pay. Once adoption scales, introducing on-chain payments can become a natural next step, layered into the experience as enterprise comfort with web3 tools increases.

## Path Forward
Filecoin’s long-term "North Star" should be for both service delivery and payments to become Filecoin-native. However, incentives to promote on-chain payments should be separate from Filecoin Plus initiatives to avoid impeding adoption of storage services.

**Comment by Kevin-FF-USA** [comment]
Hi @lksbrssr,
Outstanding!  Thanks for kicking this discussion off.  
I won't be able to get this into the 17Dec Governance call Agenda today, but will share in the Allocator/Public channels for feedback (and get on the calendar for January's Governance Calls) for full community review and feedback.

**Comment by Kevin-FF-USA** [comment]
@lksbrssr - Hoped for more people to organically find this topic!  
Maybe you and I can banter here and entice others to jump in the conversation.  : )

**These are great points**. I'm aligned and also very keen to see orgs building in the ecosystem.

Mandating on-chain payments for Filecoin Plus (Fil+) storage deals has been discussed. But reading how this could lead to impeding adoption of storage services is worth reviewing.  When we last spoke on topic, these were the two points that were driving the discussion.
 
1. Trust and Transparency: On-chain payments provide a clear, immutable record of transactions, ensuring all stakeholders can verify payments and preventing disputes. Also Prevents off-chain manipulation, such as misreporting payments or inflating storage deal values without actual financial commitment.  

- One of the issues uncovered when RG was running Trust and Transparency in 2023 and 2024 was preventing Rent-Seeking & Collusion. Off-chain payments could enable side deals where parties engage in under-the-table transactions to game the system.
- Off-chain can't ensure Fil+ allocations are genuinely competitive rather than influenced by private arrangements that may prioritize personal interests over network health.  This is less of an action item to unpack now, but something we need to be mindful of.

2. On-chain payments can be automated via smart contracts, ensuring compliance with deal terms and reducing administrative overhead. 
- Eliminates reliance on trust, since payments can be programmed to execute based on provable conditions (e.g., storage deal completion, uptime SLAs). This feel more like a GOAL to reach in the long term - but want to be mindful to not create a hurdle for growth in the now.


If I'm reading the proposal as you intended, requiring on-chain payments would be an adoption hurdle for your organization and others operating in the space correct?  
- What would be an achievable verification method you would suggest?

**Comment by GLMDesigns** [comment]
Nice post. The key issue, IMHO, was mentioned here "The real debate isn’t on-chain payments vs. off-chain payments. It’s about adoption vs. non-adoption." People still think that accessing data should be as simple as using Google Drive or iCloud. I think one of the best use cases is for business that need to keep boxes of paper records. Oftentimes they put these boxes in storage and then never access them again.
